bin/delayed_job

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
+
+require File.expand_path(File.join(File.dirname(__FILE__), '..', 'config', 'environment'))
+require 'delayed/command'
+Delayed::Command.new(ARGV).daemonize
```

Comment:
What's this? :thinking:

Reply:
Its auto-generated when installing delayed_job, but we're not actually using it.

Reply:
See https://github.com/collectiveidea/delayed_job#running-jobs